### ISSUE TITLE
Seems like a bug when import InceptionV1

### ISSUE THREAD

**Issue by ShiRuiUT** [tensorflow/lucid#140]
Hi, I use tensorboard draw the graph before final optimization and find a weird thing. That is the output of softmax2_pre_activation is 1008 dimension. I don't know whether it hurt the results of attribution method. Here is the screenshot about it.
Ps: classification results of the dog and cat image are right.

![bug_image](https://user-images.githubusercontent.com/19572042/52519125-1cb94380-2c9a-11e9-84fd-695fce88905b.JPG)

**Comment by colah** [comment]
1008 is actually correct! InceptionV1 was trained with a couple extra "dummy" outputs in the softmax.
